vms/platformvm/vm_test.go

```diff
@@ -2244,7 +2341,7 @@ func TestBaseTx(t *testing.T) {
 	}
 	require.Equal(totalOutputAmt, key0OutputAmt+key1OutputAmt+changeAddrOutputAmt)
 
-	require.Equal(vm.TxFee, totalInputAmt-totalOutputAmt)
+	require.Equal(vm.Config.TxFee, totalInputAmt-totalOutputAmt) // wallet inflates baseTx fee
```

Comment:
What is this comment trying to say? afaict the wallet pays the minimum fee here... Is it trying to say something else?

Reply:
reverted- this test doesn't change in this PR